CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # webkms-client ChangeLog
 
+## 8.0.0 -
+
+### Changed
+- **BREAKING**: Update error messages, make them more specific.
+  Add `cause` property to the thrown errors, and include `requestHost` for
```

Comment:
This needs updating since `requestHost` doesn't seem to be a thing anymore.

Reply:
Fixed (changed to `requestUrl`).